### DISCUSSION TITLE
Changes to groups and application icons require logout/log in

### DISCUSSION THREAD

**Discussion by skjdghsdjgsdj** [goauthentik/authentik#12494] [Q&A]
I'm running Authentik 2024.12.1 in a `docker compose` stack. It seems any time I do the following, end users need to log out then back in for changes to take effect:

* Adding/removing users from groups. I have applications assigned to groups so this grants and removes access to applications.
* Changing an application's icon. A hard refresh doesn't matter; I have to actually log out.

Am I doing something wrong, or is there a way to configure Authentik to have these changes take effect immediately without requiring a logout/log in?

**Comment by rissson** [comment]
You can click "clear cache" on the flows page in the admin interface to avoid this
